src/go/rpk/pkg/config/config.go

```diff
@@ -57,6 +57,11 @@ func Default() *Config {
 	}
 }
 
+func ProdDefault() *Config {
+	cfg := DevDefault()
+	return setProduction(cfg)
```

Comment:
This is still opt out and we risk accidentally opting into a new dev default and not opting out of it in production.

Can we create an issu change this to a new struct initialization that returns only prod defaults, and then also in that issue track that `mode production` should change to use this method?

Reply:
https://github.com/redpanda-data/redpanda/issues/6934